src/main/java/uk/gov/hmcts/juror/api/moj/domain/Juror.java

```diff
@@ -291,6 +295,12 @@ public class Juror extends Address implements Serializable {
     @NotAudited
     private JurorResponseCommon jurorResponse;
 
+    @Column(name = "phone_number_combined")
+    @Setter(AccessLevel.NONE)
+    @NotAudited
+    @Generated
+    private String phoneNumberCombined;
```

Comment:
could we not just create a function that does this rather than create a new data item?

Reply:
If we used a function, we could not grab the field via QueryDSL apart of our search logic

Reply:
there is a possible solution to coalesce the values in queryDSL as well. but will leave that for you to decide.

Reply:
We could do but then we are duplicating the logic for getting the combined phone number.
Once in the function one in the query DSL sections